### PR TITLE
Docs: Progress Tweaks

### PR DESCRIPTION
**Pull request by dakanji**

### CODE REVIEW

File: Docs/Configuration.tex [hunk 1]
```diff
@@ -6586,7 +6582,7 @@ \subsection{ProtocolOverrides Properties}\label{uefiprotoprops}
   \texttt{HashServices}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Forcibly reinstalls Hash Services protocols with builtin
+  \textbf{Description}: Forcibly replaces Hash Services protocols with builtin
```
Comment (vit9696) [inline Docs/Configuration.tex:6585]: Forcibly is really an artefact of some old writing style. The behaviour is all the same. In my opinion, either drop `Forcibly` everywhere, or add it.

Comment (dakanji) [reply Docs/Configuration.tex:6585]: Will drop remaining examples

File: Docs/Configuration.tex [hunk 2]
```diff
@@ -1343,15 +1343,14 @@ \subsection{Patch Properties}\label{booterpropspatch}
...
   \textbf{Type}: \texttt{plist\ data}\\
   \textbf{Failsafe}: Empty\\
-  \textbf{Description}: Data to find. This must equal to \texttt{Replace} in size.
+  \textbf{Description}: Data to find. Must be equal to \texttt{Replace} in size.
 
 \item
   \texttt{Identifier}\\
   \textbf{Type}: \texttt{plist\ string}\\
-  \textbf{Failsafe}: Empty\\
+  \textbf{Failsafe}: Empty (Match any booter)\\
```
Comment (vit9696) [inline Docs/Configuration.tex:1351]: This is an overlook actually. It should be `Any` for failsafe. I fixed the code in master. Specifying an empty value is not allowed here.

Comment (dakanji) [reply Docs/Configuration.tex:1351]: OK. What should it say?

Comment (dakanji) [reply Docs/Configuration.tex:1351]: Just to be clear, do you mean:
`\textbf{Failsafe}:  \texttt{Any} (Match any booter)\\`

Comment (vit9696) [reply Docs/Configuration.tex]: Yeah, thanks

**Pull request merged**